plugins/GSdx/GSState.cpp

```diff
@@ -44,6 +44,7 @@ GSState::GSState()
 	, m_crc(0)
 	, m_options(0)
 	, m_frameskip(0)
+	, m_dump_root("")
```

Comment:
my mistake, I don't need to initialize this to an empty string.
will fix tomorrow.